webapp/src/server_fetch_utils.js

```diff
@@ -422,6 +423,23 @@ export function saveCollection(collection_id) {
     });
 }
 
+export function saveUser(user_id, user) {
+  return new Promise((resolve, reject) => {
+    fetch_patch(`${API_URL}/users/${user_id}`, user)
+      .then(function (response_json) {
+        if (response_json.status === "success") {
+          console.log("Save successful!");
+          resolve();
+        } else {
+          reject(new Error("Save unsuccessful"));
+        }
+      })
+      .catch(function (error) {
+        reject(error);
+      });
```

Comment:
Does anything happen with these errors at the moment? For some operations we return a native browser alert, which are a bit ugly, but might be helpful